Electron-Basic-Video-Chat/openTok.js

```diff
@@ -0,0 +1,39 @@
+//Set Credentials
+const apiKey = "";
+const sessionId = "";
+const token = "";
+
+//Initialize Session
+const session = OT.initSession(apiKey, sessionId);
+
+//Set session event listeners
+session.on({
+  streamCreated: (event) => {
+    session.subscribe(event.stream, 'subscriber', (error) => {
+      if (error) console.log(`There was an issue subscribing to the stream ${event}`);
```

Comment:
`if` brace style should be consistent - if you do one-liners, use one-liners always when possible. If you don't, always use brackets. There's some file-wide variation that can be confusing for someone who is unfamiliar with the language.

IMHO I would recommend brackets always - the default ESLint style can be found here: https://eslint.org/docs/rules/brace-style#options
e.g.
```
if (cond) {
  action();
}
```

Reply:
fixed in the latest commit